docker/model-server/release.sh

```diff
@@ -11,17 +11,34 @@ fi
 GIT_ROOT=$(git rev-parse --show-toplevel)
 cd "$GIT_ROOT"/docker/model-server
 
-echo "Releasing debian based docker base image.."
-docker build --pull \
-    --build-arg BENTOML_VERSION="$BENTOML_VERSION" \
-    -t bentoml/model-server:"$BENTOML_VERSION" \
-    -t bentoml/model-server:latest \
-    .
-docker push bentoml/model-server:"$BENTOML_VERSION"
+PYTHON_MAJOR_VERSIONS=(3.6 3.7 3.8)
+#which version to tag with latest tag
+PYTHON_LATEST_VERSION=3.7
+
+
+echo "Building debian based docker base images for ${PYTHON_MAJOR_VERSIONS[*]}"
```

Comment:
Looks like I should release these images for 0.8.6 in order for the API server integration tests in CI to pass, I will go ahead and do that and re-trigger the tests